ui/package.json

```diff
@@ -48,7 +48,8 @@
     "npm-link-shared": "^0.5.6",
     "react-app-rewire-alias": "^1.1.7",
     "react-dev-utils": "^12.0.0",
-    "recoil": "^0.5.2"
+    "recoil": "^0.5.2",
+    "react-error-overlay": "6.0.9"
```

Comment:
where did we used it?

Reply:
it's fixing the iframe bug we had

Reply:
Rami merged this only for the enterprise repo